### PR TITLE
Refactor/remove useless code

### PR DESCRIPTION
**Pull request by lastMove**
Since `ColorType` has a `color: UIColor` property, we can just use it.
To be merged _after_ #330

### CODE REVIEW

**Review by tbaranes — APPROVED** [review]

File: IBAnimatable/InteractiveGestureType.swift [hunk 1]
```diff
@@ -57,7 +57,7 @@ extension InteractiveGestureType: IBEnum {
     case "pan":
       let direction = GestureDirection(raw: params[safe: 0], defaultValue: .left)
       self = .pan(from: direction)
-    case "screenedgePan":
+    case "screenedgepan":
```
Comment (JakeLin) [inline IBAnimatable/InteractiveGestureType.swift:60]: Good find.

**Review by JakeLin — APPROVED** [review]
Well done 👍👍👍

**Comment by tbaranes** [comment]
I think you can merge even with travis failing, since travis-ci/push passed, it should be ok! Seems like it has some issues with the new xcodeproj renaming

**Comment by JakeLin** [comment]
@tbaranes it is some weird issue with Travis, I can see buddybuild passes https://dashboard.buddybuild.com/apps/56abf6d42c882e010057b182/build/57df97086c21af010069aae3

**Pull request merged**